internal/controller/apiextensions/composite/reconciler_test.go

```diff
@@ -775,7 +775,13 @@ func WantComposite(t *testing.T, want resource.Composite) func(ctx context.Conte
 		// Normally we use a custom Equal method on conditions to ignore the
 		// lastTransitionTime, but we may be using unstructured types here where
 		// the conditions are just a map[string]any.
-		diff := cmp.Diff(want, got, cmpopts.EquateApproxTime(3*time.Second))
+		diff := cmp.Diff(want, got, cmpopts.AcyclicTransformer("StringToTime", func(s string) any {
+			ts, err := time.Parse(time.RFC3339, s)
+			if err != nil {
+				return s
+			}
+			return ts
+		}), cmpopts.EquateApproxTime(3*time.Second))
```

Comment:
Is the approx time here intentionally less than the other occurrence? (30 seconds vs 3 seconds)

Reply:
Not sure about that tbh, I don't think there was a particular reason, and then we had a 99.9% success with 0s approximation so far, so I guess 3s could be enough 🤔

Reply:
@phisco makes sense -- I know @negz mentioned on the PR where I added the approx time that `30s` felt like a lot anyway. Can you update to standardize on `3s` in all locations?

Reply:
Done!